lib/default.nix

```diff
@@ -5,7 +5,7 @@ nix-std: {
   buildCosmwasmContract = let
     target = "wasm32-unknown-unknown";
     # from https://github.com/CosmWasm/rust-optimizer/blob/main/Dockerfile
-    rust = pkgs.rust-bin.stable."1.70.0".default.override {
+    rust = pkgs.rust-bin.stable."1.75.0".default.override {
```

Comment:
Does this need a pinned rust version, or could we just use nightly or `pkgs.rust-bin.stable.latest.default`?

Reply:
yes. wasm has very subtle build and runtime ssues regarding wasm features.
the best option to build with exactly same version official docker container builds.